README.md

```diff
@@ -0,0 +1,123 @@
+# ComputeCpp SDK Readme
+
+![ComputeCpp™](https://www.codeplay.com/public/uploaded/public/computecpp_logo.png)
+
+## Introduction
+---
+[![Build Status](https://travis-ci.org/codeplaysoftware/computecpp-sdk.svg?branch=master)](https://travis-ci.org/codeplaysoftware/computecpp-sdk)
+
+This is the README for the ComputeCpp SDK, a collection of sample code,
+utilities and tools for Codeplay’s ComputeCpp, an implementation of the
+SYCL programming standard. You can find more information at:
+
+* http://computecpp.codeplay.com
+* http://sycl.tech
+
+## Contents
+---
```

Comment:
This is a line break, and I don't think you want a line break immediately after a title. This appears on a few, it would be better to remove them all